lib/std/Target.zig

```diff
@@ -740,6 +750,7 @@ pub const arm = @import("Target/arm.zig");
 pub const avr = @import("Target/avr.zig");
 pub const bpf = @import("Target/bpf.zig");
 pub const csky = @import("Target/csky.zig");
+pub const ez80 = @import("Target/generic.zig");
```

Comment:
```suggestion
pub const z80 = @import("Target/generic.zig");
```

This needs to match the `Cpu.Arch.Family` name.